### PR TITLE
Chapter 3 : openID and auth

### PR DESCRIPTION
**Pull request by mere-teresa**
Not easy to write, might need a heavy rework. Reviews are more than welcome.

### CODE REVIEW

File: doc_sphinx/tutorials/3_openid.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+================
+Authentication 
+================
+
+As long as you need authentication, your application has to be registered on the BIMData Connect account manager.
```
Comment (Amoki) [inline doc_sphinx/tutorials/3_openid.rst]: They will always need authentication, we don't allow any unauthenticated call

File: doc_sphinx/tutorials/3_openid.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+What's an Access Token ?
+=========================
+
+The Client ID and the Client Secret are the 2 mandatory elements to get an Access Token from the Authentication Server.
+You will need this Access Token for every call of the BIMData's API.
```
Comment (Amoki) [inline doc_sphinx/tutorials/3_openid.rst]: "call to BIMData API" ?

File: doc_sphinx/tutorials/3_openid.rst (hunk 1 above)
Comment (mere-teresa) [reply doc_sphinx/tutorials/3_openid.rst]: Then "your app needs to be registered?"

Comment (Amoki) [reply doc_sphinx/tutorials/3_openid.rst]: :+1:

**Pull request merged**